apps/condo/pages/ticket/[id]/pdf.tsx

```diff
@@ -180,11 +180,15 @@ const PdfView = () => {
                             <Typography.Text ellipsis>{get(ticket, ['createdBy', 'name'])}</Typography.Text>
                         </Typography.Text>
                     </Col>
-                    <Col span={8}>
+                    <Col span={10}>
                         {isEmergency && (<Typography.Title level={2}>{EmergencyMessage.toLowerCase()}</Typography.Title>)}
                         {isWarranty && (<Typography.Title style={ticketLabelStyle} level={2}>{WarrantyMessage.toLowerCase()}</Typography.Title>)}
                         {isPaid && (<Typography.Title style={ticketLabelStyle} level={2}>{PaidMessage.toLowerCase()}</Typography.Title>)}
-                        {isReturned && (<Typography.Title style={ticketLabelStyle} level={2}>{ReturnedMessage.toLowerCase()}</Typography.Title>)}
+                        {statusReopenedCounter > 0 && (
+                            <Typography.Title style={ticketLabelStyle} level={2}>
+                                {`${ReturnedMessage.toLowerCase()} ${statusReopenedCounter > 1 ? statusReopenedCounter : ''}`}
```

Comment:
It’s strange that you use lover here. Change the translation or use another message id

Reply:
Messages here are in lower case. I don't know why, but when adding a new one, everyone uses lowercase (see 2 lines above)